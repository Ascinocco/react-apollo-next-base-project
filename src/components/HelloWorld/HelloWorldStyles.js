import styled from 'styled-components';

export default styled.div`
  background-color: black;
  color: white;
  width: 250px;
  /* @TODO: Anthony - this should be causing linting to fail */
  height: 250px;
`;
