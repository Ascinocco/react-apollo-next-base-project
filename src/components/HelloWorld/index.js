import React from 'react';
import HelloWorldStyles from './HelloWorldStyles';

const HelloWorld = () => (
  <HelloWorldStyles>
    <p>Hello World</p>
  </HelloWorldStyles>
);

export default HelloWorld;
