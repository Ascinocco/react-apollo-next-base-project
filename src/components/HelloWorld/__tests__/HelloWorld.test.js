import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

// @TODO: Anthony - out of order import should be causing tests to fail
import HelloWorld from '../index';

describe('<HelloWorld/>', () => {
  it('renders and matches the snapshot', () => {
    const wrapper = shallow(<HelloWorld />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
