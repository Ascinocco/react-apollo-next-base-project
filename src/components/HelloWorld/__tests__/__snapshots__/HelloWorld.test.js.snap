// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HelloWorld/> renders and matches the snapshot 1`] = `
<HelloWorldStyles>
  <p>
    Hello World
  </p>
</HelloWorldStyles>
`;
